chicago/templates/chicago/about.html

```diff
@@ -115,19 +115,30 @@ <h3 id='tags'>Legislation tags</h3>
 
         <h3>Data</h3>
 
-        <p>The data from this website comes from the <a href="http://chicago.legistar.com/" target="_blank">Chicago City Council legislation site</a>, a system built by <a href='http://www.granicus.com/' target="_blank">Granicus</a> using their <a href='https://www.granicus.com/solutions/meeting-agenda-suite/' target="_blank">Legistar Legislative Management Suite</a>. The data is managed by the <a href='http://chicityclerk.com/' target="_blank">Office of the City Clerk</a>. On Councilmatic, each piece of legislation provides a link to its source for reference.</p>
+        <p>The data on this website comes from the <a href="http://chicago.legistar.com/" target="_blank">Chicago City Council legislation site</a>, a system built by <a href='http://www.granicus.com/' target="_blank">Granicus</a> using their <a href='https://www.granicus.com/solutions/meeting-agenda-suite/' target="_blank">Legistar Legislative Management Suite</a>. The <a href='http://chicityclerk.com/' target="_blank">Office of the City Clerk</a>manages the data, and on Councilmatic, each piece of legislation provides a link to its source for reference.</p>
 
-        <p>This data is then collected daily with the help of the <a href='http://webapi.legistar.com/' target="_blank">Legistar Web API</a> and stored using the <a href="http://opencivicdata.org/about/" target="_blank">Open Civic Data</a> standard and platform. Open Civic Data standardizes information about people, organizations, events and bills at any level of government and was built in collaboration with <a href="http://sunlightfoundation.com/" target="_blank">The Sunlight Foundation</a>, <a href="https://developers.google.com/civic-information/?hl=en" target="_blank">Google</a>, <a href="http://www.granicus.com/" target="_blank">Granicus</a> and <a href="http://www.opennorth.ca/" target="_blank">Open North</a>.
-        </p>
+        <p>Daily, DataMade collects data from Legistar and the <a href='http://webapi.legistar.com/' target="_blank">Legistar Web API</a>, which we store using the <a href="http://opencivicdata.org/about/" target="_blank">Open Civic Data</a> standard and platform. Built in collaboration with <a href="http://sunlightfoundation.com/" target="_blank">The Sunlight Foundation</a>, <a href="https://developers.google.com/civic-information/?hl=en" target="_blank">Google</a>, <a href="http://www.granicus.com/" target="_blank">Granicus</a>, and <a href="http://www.opennorth.ca/" target="_blank">Open North</a>, Open Civic Data standardizes information about people, organizations, events, and bills at any level of government.</p>
 
         <p><a href='http://opencivicdata.org' target="_blank"><img style='height:100px;' src='/static/images/ocd.jpg' /></a></p>
+
+        <p>Anyone can use the OCD API! This searchable government-transparency tool conatins several useful endpoints:</p>
+        <ul>
+            <li><a href="https://ocd.datamade.us/bills/" target="_blank">bills</a></li>
```

Comment:
lets add a basic description to each of these, like:

* **bills** - Legislation introduced to City Council and their status and sponsors
* **events** - Past and upcoming Council and Committee meetings